tests/kas_with_instance.spec.ts

```diff
@@ -183,3 +183,75 @@ test('create and delete a Kafka Topic', async ({ page }) => {
   await createKafkaTopic(page, testTopicName);
   await deleteKafkaTopic(page, testTopicName);
 });
+
+// test_4kafka.py test_edit_topic_properties_after_creation
+test('edit topic properties after creation', async ({ page }) => {
+  await navigateToKafkaTopicsList(page, testInstanceName);
+  await createKafkaTopic(page, testTopicName);
+
+  const instanceLinkSelector = page.getByText(testTopicName);
+  const row = page.locator('tr', { has: instanceLinkSelector });
+
+  await row.locator('[aria-label="Actions"]').click();
+  await page.getByText('Edit topic configuration').click();
+
+  // store previous num partitions value to compare afterwards
+  const numPartitionsBefore: string = await page.locator('input[name="num-partitions"]').getAttribute('value');
+
+  for (let i = 0; i < 2; i++) {
+    await page.locator('button[name="num-partitions"]').nth(1).click();
+  }
+  await page.locator('button[name="num-partitions"]').first().click();
```

Comment:
I am not sure if I follow the steps. What exactly does this do?

Reply:
so you click on the small buttons available in the field, increase it to 2 and then decrease it? If yes, this deserves a comment

Reply:
Also, you should probably check the value

Reply:
values are checked at the bottom of the test

![image](https://user-images.githubusercontent.com/100594859/213181776-c3d9c14b-1ec4-4700-8b79-b994cd98b6dc.png)

Reply:
but you increase the value (line 202) and right after decreasing it (line 204), you should check the value between. Or am I missing something?

Reply:
in the end it is just changing the number from default value, but on the same test we make sure the + and - are working. We increase by 2 and decrease by 1

Reply:
ok, I guess I can live with it. You can reuse the code via function as it is same as bellow. Also wouldn't be better to use `nth(0)` instead of `.first()` ?

Reply:
Replaced for `await numPartitionsButton.nth(0).click();`